libs/@local/hash-graph-sdk/python/graph_sdk/filter/base.py

```diff
@@ -0,0 +1,69 @@
+from abc import ABC
+from typing import Generic, Self, TypeVar
+
+from graph_client import QueryToken
+from graph_client.models import Selector
+
+from graph_sdk.query import Path
+
+
+class AbstractPath(ABC):
+    """An abstract path."""
+
+    path: Path
+
+    def __init__(self) -> None:
+        """Initialize the filter with a path."""
+        self.path = Path()
+
+    @classmethod
+    def from_list(cls, value: list[QueryToken]) -> Self:
```

Comment:
Maybe `from_ffi`?

Reply:
renamed in [6461db7](https://github.com/hashintel/hash/pull/2878/commits/6461db730dcc8a379540fd8b5236214b22a0ce3a)